components/kyma-environment-broker/internal/storage/dbsession/write.go

```diff
@@ -72,6 +72,14 @@ func (ws writeSession) InsertInstance(instance internal.Instance) dberr.Error {
 	return nil
 }
 
+func (ws writeSession) DeleteInstance(instanceID string) dberr.Error {
+	_, err := ws.deleteFrom(postsql.InstancesTableName).
+		Where(dbr.Eq("instance_id", instanceID)).
+		Exec()
+
+	return dberr.Internal("Failed to delete record from Instance table: %s", err)
```

Comment:
we should not return error if record was successfully deleted for storage
```suggestion
	if err != nil {
		return dberr.Internal("Failed to delete record from Instance table: %s", err)
	}
	return nil
```

Reply:
it is a mistake, of course you are right